.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,9 +14,11 @@ Contributors guide: https://github.com/electron/electron/blob/master/CONTRIBUTIN
 - [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
 - [ ] relevant documentation is changed or added
 - [ ] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
+- [ ] PR release notes describe the change in a way relevant to app-developers
 
 
 #### Release Notes
-<!-- Used to describe release notes for future release versions. Use `no-notes` to indicate no user-facing changes. -->
 
-Notes: <!-- One-line Change Summary Here-->
\ No newline at end of file
+<!-- The note that you write here will be communicated to Electron app developers in the release notes when this change is released. Write something short but descriptive (one sentence to a couple paragraphs, if appropriate) that explains what, if anything, this change means for a developer building an app with Electron. Use `no-notes` for things like chores, internal fixes, and the like. -->
```

Comment:
This is probably that I've been in OSS too long but I worry that people will tune out when they see a paragraph-long description like that.

I *really* like the idea of clarifying the template like this but suspect we'll get better results from contributors if we KISS as much as possible
```suggestion
 <!-- A one-line description for the release notes. Please make this understandable to end users. See https://github.com/electron/clerk/blob/master/README.md#examples for examples and help on special cases.
```